src/renderers/native/ReactNative/ReactNativeMount.js

```diff
@@ -0,0 +1,270 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactNativeMount
+ * @flow
+ */
+'use strict';
+
+var ReactElement = require('ReactElement');
+var ReactNativeTagHandles = require('ReactNativeTagHandles');
+var ReactPerf = require('ReactPerf');
+var ReactReconciler = require('ReactReconciler');
+var ReactUpdateQueue = require('ReactUpdateQueue');
+var ReactUpdates = require('ReactUpdates');
+var UIManager = require('UIManager');
```

Comment:
`UIManager` module doesn't exist. This will almost certainly be broken. I'm guessing there are a bunch of others in the same boat (presumably modules that aren't brought over from RN). We need to whitelist all of these in the map - https://github.com/facebook/react/blob/master/gulpfile.js#L33 (`map` is a key:value mapping, `map: Object.assign({}, require('fbjs/module-map'), {UIManager: 'UIManager', etc})`